### PR TITLE
Updated home page document

### PR DESCRIPTION
**Pull request by chdattat**
@sjimenez77, @jambulud,

I have updated the ASCII document for the home page. Please review it and let me know in case any issue.

Thanks,
Dattu Chillal

### CODE REVIEW

File: documentation/home-page.asciidoc [hunk 1]
```diff
@@ -18,14 +18,65 @@ toc::[]
...
 
-== Topbar
+== Topbar +
+This section is at the top of the page, it contains *Devon Instance* dropdown to select devonfw-ide that can be used as a base for the projects.
 
-In the bar at the bottom of the page, there is a selector where you can use the devonfw-ide that you want to use as base for your projects.
+image::images/home_page/toolbar.png["Toolbar", width="600", link="images/home_page/toolbar.png"]
 
-== Sidebar
+Next to the *Devon Instance* dropdown, there is a *quick help* icon, clicking on it will open a popup which gives some tips for how to use Devon Dashboard IDE.
```
Comment (sjimenez77) [inline documentation/home-page.asciidoc]: Please, fix all the **devonfw** names in this PR and in other documentation files. Please, remember that it is not Devon, devon, etc. It is just **devonfw** in lowercase. Thanks!

**Review by sjimenez77 — CHANGES_REQUESTED** [review]
The PR is fine for me. Please @chdattat, fix the requested changes and I will merge it. Thanks!

File: documentation/home-page.asciidoc (hunk 1 above)
Comment (chdattat) [reply documentation/home-page.asciidoc]: Hi @sjimenez77,

I updated the changes. Please check it.

Thanks.

Comment (sjimenez77) [reply documentation/home-page.asciidoc]: Ok, thanks.

**Review by sjimenez77 — APPROVED** [review]

**Pull request merged**